### DISCUSSION TITLE
2023년 05월 02일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#50] [General]
# 2023년 05월 02일

LOG 확실히 감기가 걸렸다. 이번에는 오래 안 아프고 싶어서 바로 병원에 갔다. 사실 병원에 안 가도 괜찮을 것 같다는 생각을 잠깐 했는데, 기침을 하다보니 피맛이 나서 ㅋㅋ 그냥 병원에 갔다. 병명은 원인불명의 기관지염. 기관지염 뭘까.. 자꾸 …

https://cyma-s.github.io/2023-05-02/

**Comment by Choi-JJunho** [comment]
인간말종 베로의 삶을 응원합니다~^^

**Reply by Cyma-s** [reply]
아 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by kevstevie** [comment]
덕분에 오늘 루틴 시작이 좋네요 감사합니다

**Reply by Cyma-s** [reply]
오늘 루틴 깨지셨겠네요
